### ISSUE TITLE
Suggestion QoL: Hotkey naming.

### ISSUE THREAD

**Issue by JohnDanielScanlon** [esjeon/krohnkite#85]
I've run into the issue of not understanding the keybinds' naming, because of the ambiguity with:

Move U/D/L/R
Up/Prev
Down/Next
Increase/Decrease
etc. 

Coupled with the issue of initially installing this from the KDE store, I was unable to understand why the hotkeys were not changing the window focus as described, as I had never used dwm. I honestly still don't understand how the dwm hotkeys function, as they don't seem to work in quarter layout, but work in column layout.

I believe that using these terms would be better for the global shortcuts:

Focus Left
Focus Right 
Focus Up / DWM function
Focus Down / DWM function 

It may also be a good idea to rename keybinds just for the sake of grouping them together. 

Krohnkite: Layout - Next
Krohnkite: Layout - Previous
Krohnkite: Layout - Monocle 
Krohnkite: Layout - Floating
Krohnkite: Resize - Left
Krohnkite: Resize - Grow
etc for each type of bind. 

Thanks!

**Comment by esjeon** [comment]
I agree. There has been lots of afterthoughts, which causes some consistency issues w/ the terminology.

However, the thing is that KDE identifies shortcuts by *name*, and everyone must rebind shortcuts whenever the name changes. Maybe I can provide a migration tool, but it still require manual intervention from users.

So, this should be done only when a major version up is made, which I think would happen within several months.
